R/area_of_applicability.R

```diff
@@ -142,6 +142,11 @@ ww_area_of_applicability.rset <- function(x, y = NULL, importance, ...) {
 
   if (missing(y) || identical(y, NULL) || identical(y, NA)) y <- NA_real_
 
+  # Standardize ALL data following CAST
+  res <- standardize_and_weight(
```

Comment:
don't think this does what you want -- now _only_ the first fold's data is going to be standardized and weighted, none of the other folds